accelerators/federated-learning/automated_provisioning/ps/Utils.ps1

```diff
@@ -0,0 +1,41 @@
+function Wait-For-Successful-Registration(){
```

Comment:
Nit: `utils` and `helpers` are not great things to name files. Something like `AzureUtilities` is a bit better.